spec/rubocop/cop/style/option_hash_spec.rb

```diff
@@ -2,11 +2,9 @@
 
 require 'spec_helper'
 
-describe RuboCop::Cop::Style::OptionHash do
-  subject(:cop) { described_class.new }
-  before(:each) do
-    inspect_source(cop, source)
-  end
+describe RuboCop::Cop::Style::OptionHash, :config do
+  subject(:cop) { described_class.new(config) }
+  let(:cop_config) { { 'Blacklist' => ['options'] } }
```

Comment:
Isn't that what the `let(:cop_config)` block and the test on line 49 is? was there another variant that you wanted?